src/interfaces/IUniswapV3Pool.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: BUSL 1.1
```

Comment:
I think we can just install the v3-core via forge and import the interface, and it's not BUSL its GPL 2 iirc